stores/flatten.go

```diff
@@ -43,10 +44,10 @@ func flattenValue(value interface{}) interface{} {
 	return output
 }
 
-// Flatten flattens a map with potentially nested maps into a flat
+// flattenMap flattens a map with potentially nested maps into a flat
 // map. Only string keys are allowed on both the top-level map and
 // child maps.
-func Flatten(in map[string]interface{}) map[string]interface{} {
+func flattenMap(in map[string]interface{}) map[string]interface{} {
```

Comment:
IMO this funcion should not be renamed.

Reply:
Do we still want to leave the functions exported (capital function name)? There are no callers outside of the package.

Reply:
done